packages/loader/container-loader/src/container.ts

```diff
@@ -930,11 +930,15 @@ export class Container
 			(this.isInteractiveClient &&
 				this.mc.config.getBoolean("Fluid.Container.enableOfflineLoad")) ??
 			options.enableOfflineLoad === true;
+		const supportGetSnapshotApi: boolean =
+			this.mc.config.getBoolean("Fluid.Container.UseLoadingGroupIdForSnapshotFetch") ===
+				true && this.service?.policies?.supportGetSnapshotApi === true;
```

Comment:
Is the service instantiated at this point? I believe this will always return false.

Reply:
Ohhhh, I see

Reply:
Fixed